### PR TITLE
[Polkadot Wiki Migration] Thousand Validators Programme

### PR DESCRIPTION
**Pull request by CrackTheCode016**
> This page was migrated from https://wiki.polkadot.network/docs/thousand-validators

### CODE REVIEW

**Comment by DrW3RK** [comment]
The program is being rebranded and has new rules FYI https://docs.google.com/document/d/1TC5CHywEohV2BT4i1UFOV7ig9rwjTrSrYZr07kNvsLs/edit#heading=h.sp1cecwuwmp4

**Comment by lovelaced** [comment]
As far as I know, the 1kv program has been completely redone and renamed, and is something now called "Decentralized Nodes" and operates completely differently, so this entire page should likely be removed/rewritten by someone from W3F.

File: infrastructure/validators/thousand-validators-program.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+2. Further decentralize the validator active set.
+
+## How it Works
+
+The [nominating backend](https://github.com/w3f/1k-validators-be) will routinely change its
+nominations at every era for Polkadot, and every four eras for Kusama. The backend does
+this by short-listing candidates by validity and then sorts validators by their weighted score in
+descending order.
+
+- Validators with a higher weighted score are selected for any possible slots. As validators are
```
[the remark this answers is not in the document]
Comment (RemyLeBerre) [reply infrastructure/validators/thousand-validators-program.md:20]: Start of sentence, should be capital V.

File: infrastructure/validators/thousand-validators-program.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  things time to resolve; if they don't, leave a message in the dedicated matrix room or open an
+  issue on GitHub.
+- _The 1000 validators website is not up to date?_ See [these resources](#resources) for more
+  up-to-date information.
+
+## Resources
+
+To extract and display the 1KV Programme scores there are various tools listed below.
+
+| Resource                                                                                                                                    | Github source                                                                               | Info                                                                                                                           |
```
[the remark this answers is not in the document]
Comment (RemyLeBerre) [reply infrastructure/validators/thousand-validators-program.md:208]: Should be 'GitHub'.

File: infrastructure/validators/thousand-validators-program.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  - No more than 15% commission
+  - Have a staking proxy set up
+  - Must be on the latest release
+  - Max two nodes (under same sub/super identity)
+  - Validators must operate nodes themselves. They may not be operated by third parties or staking
+    providers.
+
+#### Nominators
+
+The below addresses are the stash / staking proxy pairs for the primary nominators the Thousand Validators programme. They are formatted like "`stash` / `staking proxy`".
```
[the remark this answers is not in the document]
Comment (RemyLeBerre) [reply infrastructure/validators/thousand-validators-program.md:76]: Very small edit here, just on a grammar sense. 

Should be: "'stash' / 'staking proxy'."

File: infrastructure/validators/thousand-validators-program.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (RemyLeBerre) [reply infrastructure/validators/thousand-validators-program.md:208]: Should be 'GitHub'

File: infrastructure/validators/thousand-validators-program.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+| Resource                                                                                                                                    | Github source                                                                               | Info                                                                                                                           |
+| ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| SubVT telegram bot for [Polkadot](https://t.me/subvt_polkadot_bot){target=\_blank} and [Kusama](https://t.me/subvt_kusama_bot){target=\_blank}                              | [Github](https://github.com/helikon-labs/subvt-backend/tree/development/subvt-telegram-bot){target=\_blank} | Telegram bot with overviews and alerts for various Polkadot and Kusama related events, including 1KV events.                   |
```
[the remark this answers is not in the document]
Comment (RemyLeBerre) [reply infrastructure/validators/thousand-validators-program.md:210]: Should be 'GitHub'.

[the remark this answers is not in the document]
Comment (RemyLeBerre) [reply infrastructure/validators/thousand-validators-program.md:210]: Should be 'GitHub'.

**Comment by eshaben** [comment]
Closing this since the program has been replaced with the Decentralized Nodes Program

**Pull request closed without merging**